MAUI/Autocomplete/UI-Customization.md

```diff
@@ -504,6 +504,121 @@ autocomplete.DropDownItemHeight = 80;
 
 ![.NET MAUI Autocomplete DropDown Item Height](Images/UICustomization/DropDownItemHeight.png)
 
+### Customize in DropDownPlacement
+
+The drop-down that shows the filtered items will be placed automatically based on the available space and can also be customized using the `DropDownPlacement` property.
+
+*   `Top` - Drop-down will be placed above the text box.
+
+*   `Bottom` - Drop-down will be placed below the text box.
+
+*   `Auto` - Drop-down will be placed based on the available space either top or bottom of the text box.
+
+*   `None` - Drop-down will not be shown with the filtered items.
+
+
+{% tabs %}
+{% highlight xaml %}
+
+ <editors:SfAutocomplete x:Name="autocomplete"
+                         WidthRequest="300"                  
+                         ItemsSource="{Binding SocialMedias}"
+                         DropDownPlacement="Top"
+                         TextMemberPath="Name"
```

Comment:
No need to mention the below properties,

 TextMemberPath="Name"
 DisplayMemberPath="Name"

Reply:
Updated changes